### DISCUSSION TITLE
Popup as default mode

### DISCUSSION THREAD

**Discussion by ljs2905** [Authenticator-Extension/Authenticator#1028] [General]
How can I set up popup as the default mode?

**Comment by Sneezry** [comment]
Does **Add to context menu** meets your use case? By enabling that option, there will be **Authenticator** item in your context menu, and when you click it, a popup window shows instead of the default mode.

**Reply by Sneezry** [reply]
![image](https://user-images.githubusercontent.com/1621293/233253087-d855b807-29a1-44b5-8409-0b2522d2db6b.png)

![image](https://user-images.githubusercontent.com/1621293/233253121-50f03298-ffea-44cd-bd85-a4b732c16bc0.png)

**Reply by lsloan** [reply]
If Authenticator is activated via contextual menu for an input field, is there any way it could **_paste_** the value into that input field when it's selected instead of copying it?  Since we'd need to paste it anyway, that could save us a step.

**Comment by ljs2905** [comment]
Thanks a lot. Enviado do Email para Windows De: Zhe LiEnviado:20 April 2023 00:47Para: Authenticator-Extension/AuthenticatorCc:ljs2905; AuthorAssunto: Re: [Authenticator-Extension/Authenticator] Popup as default mode (Discussion #1028) —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
